packages/google_maps_flutter/android/src/main/java/io/flutter/plugins/googlemaps/GoogleMapController.java

```diff
@@ -399,4 +406,17 @@ public void setMinMaxZoomPreference(Float min, Float max) {
   public void setZoomGesturesEnabled(boolean zoomGesturesEnabled) {
     googleMap.getUiSettings().setZoomGesturesEnabled(zoomGesturesEnabled);
   }
+
+  @Override
+  public void setMyLocationEnabled(boolean myLocationEnabled) {
+    this.myLocationEnabled = myLocationEnabled;
+    if (googleMap != null) {
```

Comment:
When will this be null here?

Reply:
Yeah my solution is unclear. In `GoogleMapBuilder` I needed a way to pass the `myLocationEnabled` boolean to the `GoogleMapController` for map initiation, because, like `trackCameraPosition`, it isn't a parameter in the `GoogleMapOptions` class. I was calling the `setMyLocationEnabled` function to just set the `myLocationEnabled` boolean so the map could access it once it was created, but this is kind of out of order.

In the commit I have coming I'm just passing the boolean into the `GoogleMapController` constructor. Now it can access the boolean during `onMapReady()`. This is necessary because it is needed right away during initiation, rather than trackCamera which is used only during subsequent calls.